libs/ktem/flowsettings.py

```diff
@@ -32,7 +32,15 @@
     "path": str(user_cache_dir / "vectorstore"),
 }
 KH_LLMS = {}
-KH_EMBEDDINGS = {}
+KH_EMBEDDINGS = {
+    "local-mxbai-large-v1": {
+        "spec": {
+            "__type__": "kotaemon.embeddings.FastEmbedEmbeddings",
+            "model_name": "mixedbread-ai/mxbai-embed-large-v1",
+        },
+        "default": True,
+    }
+}
```

Comment:
currently models can still be configured using the .env file, so could you move this to the end after checking  `KH_EMBEDDINGS ` being empty ?

Reply:
I just moved the local declaration into below. Please check @lone17